nominees/Women’s issues

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Women’s issues within political party platforms",
+  "description": "A data story about how often women's issues appeared in the text within political party platforms",
+  "website": "https://pudding.cool/projects/votes-for-women/",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/the-pudding/votes-for-women/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 5,
+      "evidenceText": "This project explores every political party platform between 1840 and 2016 to find how often women's issues appeared in the text. 45 documents. 934,068 total words. 28,854 words related to women.",
+      "evidenceURL": "https://pudding.cool/projects/votes-for-women/, https://github.com/the-pudding/votes-for-women"
```

Comment:
@Rolikasi in as much as possible for the `evidenceURL`, try to point to `blog` or `news articles` and avoid `Github repos`. Kindly remove the `Github repo` saved under the `evidenceURL`. Thanks.